build problem on 5.12

```
  CC [M]  /var/lib/dkms/joydev_dance/1.0/build/joydev_dance/joydev_dance.o
/var/lib/dkms/joydev_dance/1.0/build/joydev_dance/joydev_dance.c:29:25: error: expected declaration specifiers or ‘...’ before string constant
   29 | MODULE_SUPPORTED_DEVICE("input/js");
      |                         ^~~~~~~~~~
```

--- a/dkms/joydev_dance-1.0/joydev_dance/joydev_dance.c
+++ b/dkms/joydev_dance-1.0/joydev_dance/joydev_dance.c
@@ -26,7 +26,6 @@
 
 MODULE_AUTHOR("Vojtech Pavlik <vojtech@ucw.cz>, Adiel Mittmann <adiel@mittmann.net.br>");
 MODULE_DESCRIPTION("Joystick device interfaces (patched with axis fix for dance pad)");
-MODULE_SUPPORTED_DEVICE("input/js");
 MODULE_LICENSE("GPL");
 
 #define JOYDEV_MINOR_BASE	0

